Makefile

```diff
@@ -17,7 +17,22 @@ create_database: .env
 	docker compose run server create_db
 
 clean:
-	docker compose down && docker compose rm
+	docker compose down
+	docker compose --project-name cypress down
+	docker compose rm --stop --force
+	docker compose --project-name cypress rm --stop --force
+	docker image rm --force \
+		cypress-server:latest cypress-worker:latest cypress-scheduler:latest \
+		redash-server:latest redash-worker:latest redash-scheduler:latest
+	docker container prune --force
+	docker image prune --force
+	docker volume prune --force
+
+clean-all:
```

Comment:
You'd want to add this target to the `.PHONY` section... most of the rules in this makefile do not create a target file.

Then add it as a dependency

```
clean-all: clean
```

Or maybe not? I wouldn't mind typing `make clean; make clean-all`

Reply:
Thanks @eradman.  I'm clearly still coming up to speed with this stuff again. :innocent:

Looking at that bit now.

Reply:
Personally I'd rather keep the `clean-all` target incorporating the `clean` one.  But I'm not even sure if we ***really*** need it. :smile:

Reply:
k, updated.  `clean-all` added to the `.PHONY` list, and `clean` added as a dependency of `clean-all` so it doesn't need embedding as `make clean`.

In testing, `make clean` is removing both the redash and cypress containers + dev images, with `make clean-all` removing them plus the associated images (redis, pgautoupgrade, last redash release).